### ISSUE TITLE
Dynamic require of "os" is not supported

### ISSUE THREAD

**Issue by max512060925** [antfu/unplugin-auto-import#405]
### Describe the bug

"unplugin-auto-import": "^0.16.5"

`pnpm run build`

show error
failed to load config from **\vite.config.ts
error during build:
Error: Dynamic require of "os" is not supported

vite.config.ts

```typescript
import { defineConfig } from 'vite'
import vue from '@vitejs/plugin-vue'
import AutoImport from 'unplugin-auto-import/vite'
// https://vitejs.dev/config/
export default defineConfig({
  plugins: [
    vue(),
    AutoImport({
      include: [/\.[tj]sx?$/, /\.vue$/, /\.vue\?vue/],
      imports: ['vue', 'vue-router', '@vueuse/core', 'pinia'],
    }),
  ],
})
```

### Reproduction

ditto

### System Info

```Shell
win11 MacOS13 node18
```

### Used Package Manager

pnpm

### Validations

- [X] Follow our [Code of Conduct](https://github.com/antfu/.github/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guide](https://github.com/antfu/contribute).
- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A, please open a GitHub Discussion instead.
- [X] The provided reproduction is a [minimal reproducible](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by theramenboy** [comment]
Same problem here. 
If I install version 0.16.4 it works again

It seems to be a problem withing package "fast-glob@3.3.0"

Follow.

**Comment by zengsun** [comment]
yes, same problem !

**Comment by zkmefun** [comment]
yes, same problem !

**Comment by linhua117** [comment]
Encounter the same problem

**Comment by 0xiaoxi** [comment]
Marker,  same problem.

**Comment by codesyofo** [comment]
![image](https://github.com/antfu/unplugin-auto-import/assets/49975575/e9af0b98-1f36-4af9-82ff-c777c73dd164)

```shell
npm i unplugin-auto-import@https://registry.npmmirror.com/unplugin-auto-import/-/unplugin-auto-import-0.16.4.tgz
```

**Comment by mdwekat** [comment]
+1
Same here, `v0.16.4` works!

**Comment by mrforwhat** [comment]
npm run build not works at 0.16.4,  but pnpm run build works!

**Comment by zatsunen** [comment]
yes, same problem !

**Comment by antfu** [comment]
Should be fixed in 0.16.6
